src/js/commands/image.js

```diff
@@ -13,6 +13,8 @@ export default class ImageCommand extends Command {
     let beforeSection = headMarker.section;
     let afterSection = beforeSection.next;
     let section = this.editor.builder.createCardSection('image');
+    // the postEditor needs to know this section's parent in order to `insertSectionBefore`
+    section.parent = beforeSection.parent;
```

Comment:
This seems wrong- adoption to a list should decide the parent. More comments below.